Delete this xd

MrZender opened the pull request.

Commit:
huashan: remove sony FM stuff

Change-Id: Ie8ef7dcb70cbe94bd11f3c1d30ca35fd109092f4

Commit:
huashan: Enable NEEDS_VECTORIMPL_SYMBOLS required for the blobs.

Change-Id: I2a3c4900ecb749b673fc5cc24ecd139b6cdf06e9

stellirin committed:
huashan: init: add updated FUSE services for storage

Change-Id: If3485b2c00eee85ffa2e1480bb2167baaa1e5f7f

stellirin committed:
huashan: init: update the WiFi services

Change-Id: I09b60d1c17ea77d0cb1086437c5bf2800331d260

stellirin committed:
huashan: overlay: move packages/Phone to services/Telephony

Change-Id: Ic8750145fab0fb1156d6df1df74263797771c6df

stellirin committed:
huashan: overlay: allow transparency on lockscreen

Change-Id: I76a87f35f96c361fd19df61bdc87a84b948ff36f

stellirin committed:
huashan: build memtrack.msm8960

Change-Id: Id11c6a2931ac1f34df49da37ad0520e09f83fd67

stellirin committed:
huashan: we have our own camera wrapper, don't build generic

Change-Id: I34e7313e1ce06c0d367d4cd16a50214d7809c23f

stellirin committed:
huashan: overlay: move packages/Camera to packages/Gallery2

Change-Id: I1b760ece868eecffbff75f360613a14e19f07d40

stellirin committed:
Huashan: fix brightness values

Our driver supports a max lux value of 1024 (like blue platform) but even
under inder intense light the sensor will only ever report a max of 1023.

Change-Id: I07a9d2da6efc7b22b1853ee2fd166597ab8b8b4d

brinlyau committed:
Update README notes for CM11 and fix pic

Change-Id: I974d3a0ba193611ae7234d6cf8201dba79f510c6

Commit:
huashan: use -caf repo

Change-Id: Iaa6a3da2815f06c86955789dedae253703a81e76

freexperia committed:
huashan: dexopt /system apps to /cache

Change-Id: I46b30ef34e79a033147aab81575770fa4d4c878a

Commit:
huashan: use -caf repo (2)

Change-Id: I4ff939e4986cf35ba632e3dbeb75bd7adafecd15

Commit:
huashan: bt: fix exit code in case of firmware download errors

Change-Id: I01d561934d01ed980cde15a8b2572350c71b7d7d

Commit:
SELinux: move policy from qcom-common

Change-Id: Ie3083fa1756648ecc622630a76115dcc4ccdabea

Commit:
Use "include" instead of "-include" wherever appropriate

* When using "-include", make ignores a makefile which does not
  exist, with no error message.
* Use "include" for all makefiles which are essential for a
  working build.

Change-Id: Ic00ba2e9246da3ec47102f79b9c2725ed155b4cf

stellirin committed:
huashan: Increase ramdisk offset due to plump kernel

Change-Id: I670775b1ad5e1a53f0b20f7d848f394c2f852c39

stellirin committed:
Remove TARGET_DISPLAY_USE_RETIRE_FENCE

New kernel does not need it

Change-Id: Ibaae37e811647eca18fbadc26573c9de21cc0582

stellirin committed:
Remove -DMR0_CAMERA_BLOB

not needed for new camera blobs

Change-Id: I60f0b6fcfc618661432c5cc75b893876302537ff

stellirin committed:
huashan: Update tad_static and wait3tad_static in /sbin

Change-Id: If1257c444e837df01dfccfca49f513be4898f707

Commit:
huashan: update bt init script

Change-Id: I7a17acbf6a939a3eceac0886453007f487cb12cf

stellirin committed:
huashan: Add irsc_util service and sec_config file

Change-Id: I4e0623e3246ac29d2f806928fe346876670db74a

stellirin committed:
huashan: add time and wifi services

Change-Id: I4b2427bedf4dd226d3c49c3c06000767f3497668

stellirin committed:
huashan: add QCOM Location services and configs

Change-Id: I683950262b193a8973133a773dd269507aff1db4

stellirin committed:
huashan: Reorganize proprietary files

great idea, intervigil

Change-Id: I106b5b017285d17e92cdc33a1a5efecb5788fd77

Commit:
huashan: Move NO_CELLINFOLIST flag to platform

older blob still need it

Change-Id: I40d392faab778a98a6eae014a6364d3311dd7e7e

stellirin committed:
huashan: Upgrade init files from Android 4.3

* Closer match to Sony layout
* Add thermal control
* Also update some other config files

Change-Id: Id3e7f03f30a89c0939799a888716164ec321dd9e

stellirin committed:
huashan: additional init scripts

Change-Id: Iac08e35b9403c4a215082c9f6067b62e6a0b79ee

stellirin committed:
Revert "huashan: Move NO_CELLINFOLIST flag to platform"

We Have New Blobs

This reverts commit 33b9c1343b37cbfdf1a345dfe1a5c00e0889374a.

Change-Id: I2d753fd4f14a8726dfe36a019d2079d211f073f5

stellirin committed:
huashan: rewrite general device overlay using stock values

Change-Id: I1c743ffdcb97f189343e3b5acfa1bb3e25319d9e

cm-gerrit commented:
Hi there;

This is an automated reply, to let you know CyanogenMod doesn't merge pull
requests submitted directly to our github.com repositories.

You can submit your change requests through our public review system,
located at http://review.cyanogenmod.org/

That review system is an installation of Google's Gerrit code review.
If you've never interacted with Gerrit before, we also suggest you take
a look at http://wiki.cyanogenmod.org/wiki/Howto:_Gerrit

Thanks!

On Tue, 08 Apr 2014 04:04:22 -0700 you wrote:

> You can merge this Pull Request by running:
> 
>   git pull https://github.com/CyanogenMod/android_device_sony_huashan cm-11.0
> 
> Or you can view, comment on it, or merge it online at:
> 
>   https://github.com/CyanogenMod/android_device_sony_huashan/pull/2
> 
> -- Commit Summary --
> - huashan: remove sony FM stuff
> - huashan: Enable NEEDS_VECTORIMPL_SYMBOLS required for the blobs.
> - huashan: init: add updated FUSE services for storage
> - huashan: init: update the WiFi services
> - huashan: overlay: move packages/Phone to services/Telephony
> - huashan: overlay: allow transparency on lockscreen
> - huashan: build memtrack.msm8960
> - huashan: we have our own camera wrapper, don&#39;t build generic
> - huashan: overlay: move packages/Camera to packages/Gallery2
> - Huashan: fix brightness values
> - Update README notes for CM11 and fix pic
> - huashan: use -caf repo
> - huashan: dexopt /system apps to /cache
> - huashan: use -caf repo (2)
> - huashan: bt: fix exit code in case of firmware download errors
> - SELinux: move policy from qcom-common
> - Use &quot;include&quot; instead of &quot;-include&quot; wherever appropriate
> - huashan: update bt init script
> - huashan: Move NO_CELLINFOLIST flag to platform
> - huashan: Upgrade init files from Android 4.3
> - huashan: Increase ramdisk offset due to plump kernel
> - Remove TARGET_DISPLAY_USE_RETIRE_FENCE
> - Remove -DMR0_CAMERA_BLOB
> - huashan: Update tad_static and wait3tad_static in /sbin
> - huashan: Add irsc_util service and sec_config file
> - huashan: add time and wifi services
> - huashan: add QCOM Location services and configs
> - huashan: additional init scripts
> - huashan: Reorganize proprietary files
> - Revert &quot;huashan: Move NO_CELLINFOLIST flag to platform&quot;
> 
> -- File Changes --
> 
> ```
> M BoardConfig.mk (65)
> M README.md (6)
> M cm.dependencies (4)
> M custombootimg.mk (4)
> M huashan.mk (73)
> M overlay/frameworks/base/core/res/res/values/config.xml (50)
> R overlay/packages/apps/Gallery2/res/values/config.xml (0)
> R overlay/packages/services/Telephony/res/values/config.xml (3)
> M proprietary-files.txt (426)
> M rootdir/cmdline.txt (2)
> M rootdir/fstab.qcom (17)
> D rootdir/init.device.rc (52)
> A rootdir/init.qcom.early_boot.sh (149)
> M rootdir/init.qcom.rc (471)
> A rootdir/init.qcom.sh (144)
> A rootdir/init.qcom.syspart_fixup.sh (65)
> A rootdir/init.sony-device.rc (26)
> A rootdir/init.sony-platform.rc (55)
> A rootdir/init.sony.rc (7)
> A rootdir/init.target.rc (143)
> M rootdir/sbin/tad_static (0)
> M rootdir/sbin/wait4tad_static (0)
> M rootdir/system/etc/gps.conf (102)
> R rootdir/system/etc/hw_config.sh (2)
> M rootdir/system/etc/init.qcom.bt.sh (52)
> M rootdir/system/etc/init.qcom.fm.sh (6)
> A rootdir/system/etc/init.qcom.post_boot.sh (421)
> A rootdir/system/etc/init.qcom.thermal_conf.sh (109)
> A rootdir/system/etc/izat.conf (26)
> A rootdir/system/etc/lowi.conf (43)
> A rootdir/system/etc/quipc.conf (21)
> A rootdir/system/etc/sap.conf (42)
> A rootdir/system/etc/sec_config (3)
> R rootdir/system/etc/thermald-8960ab.conf (1)
> M rootdir/system/etc/vold.fstab (8)
> A rootdir/system/etc/xtwifi.conf (93)
> M rootdir/system/usr/keylayout/atmel_mxt_ts.kl (4)
> M rootdir/system/usr/keylayout/cyttsp-i2c.kl (2)
> M rootdir/system/usr/keylayout/gpio-keys.kl (4)
> M rootdir/system/usr/keylayout/keypad_8960.kl (2)
> M rootdir/system/usr/keylayout/keypad_8960_liquid.kl (2)
> A rootdir/system/usr/keylayout/mhl-rcp.kl (104)
> R rootdir/system/usr/keylayout/msm8960-snd-card_Button_Jack.kl (0)
> M rootdir/system/usr/keylayout/philips_remote_ir.kl (4)
> M rootdir/system/usr/keylayout/samsung_remote_ir.kl (4)
> A rootdir/system/usr/keylayout/synaptics_rmi4_i2c.kl (29)
> M rootdir/system/usr/keylayout/ue_rf4ce_remote.kl (4)
> M rootdir/ueventd.qcom.rc (63)
> A sepolicy/bluetooth_loader.te (31)
> A sepolicy/bridge.te (17)
> A sepolicy/camera.te (22)
> A sepolicy/device.te (16)
> A sepolicy/dhcp.te (1)
> A sepolicy/domain.te (3)
> A sepolicy/drmserver.te (2)
> A sepolicy/file.te (12)
> A sepolicy/file_contexts (90)
> A sepolicy/init.te (1)
> A sepolicy/kickstart.te (51)
> A sepolicy/mac_update.te (11)
> A sepolicy/mediaserver.te (13)
> A sepolicy/mpdecision.te (20)
> A sepolicy/netmgrd.te (30)
> A sepolicy/qmux.te (19)
> A sepolicy/rild.te (11)
> A sepolicy/rmt.te (32)
> A sepolicy/surfaceflinger.te (1)
> A sepolicy/system.te (15)
> A sepolicy/te_macros (12)
> A sepolicy/tee.te (2)
> A sepolicy/thermald.te (22)
> A sepolicy/ueventd.te (3)
> A sepolicy/wpa_supplicant.te (10)
> M system.prop (37)
> ```
> 
> -- Patch Links --
> 
> https://github.com/CyanogenMod/android_device_sony_huashan/pull/2.patch
> https://github.com/CyanogenMod/android_device_sony_huashan/pull/2.diff
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/CyanogenMod/android_device_sony_huashan/pull/2

VladislavKoldobskiy committed:
huashan: Move audio_firmware_file and /data/misc/audio entry to core sepolicy.

Needed after external/sepolicy 8bebb1c5dd10f704d0f5c90b9c51ed7f3e91eb06
(Change-Id: Id36203f5bb66b5200efc1205630b5b260ef97496)

Change-Id: I479b63346fb0e714f51f7dd1f4f42eacf7044418
Signed-off-by: Chirayu Desai <cdesai@cyanogenmod.org>

stellirin committed:
huashan: improved backlight curve

The curve is now linear on a logarithmic scale. Also increases the
number of values, especially at the lower end for smoother automatic
backlight adjustments (alleviates supposed screen flicker).

Change-Id: I3f2379f5a0bc6086385098a7cf0c46f083b458cd

stellirin committed:
huashan: new Adreno needs QCOM_BSP

Also remove -DQCOM_HARDWARE as it's now automatic

Change-Id: Ic138c6abdfecacb4e93e3629df8d19633a9d9176

bieltura committed:
Update extract-files for Offline Charging

File needed, called on init.qcom.rc:
https://github.com/CyanogenMod/android_device_sony_huashan/blob/cm-11.0/rootdir/init.qcom.rc#L76

Change-Id: Iecce6c0adfacd3017055c8033060f65534d2f9be

stellirin committed:
huashan: tidy up the init scripts

Better match other Sony devices as per Kali request

Change-Id: I2046460786fb5108f236fae91fb0c03a00eadbec

stellirin committed:
huashan: update selinux policy

(cherry-pick from 10d350e06a28d21e3509338a6eddba011600a4ec in fusion3-common)

Change-Id: I01ae7d72f00553390e4625f0aea94fc17110a94d

brinlyau committed:
huashan: full_huashan - change device model name to use real retail name

Sorry about patch spam

as suggested by http://forum.xda-developers.com/showpost.php?p=52511432&postcount=780 we should correct this overide.

this patch does not affect functionality of device (just change the name identified in Android in build.prop)

Change-Id: I20fbcc99d0572488ac8d2b532fd01e29db1e098d

brinlyau committed:
huashan: cleanup a makefile pointing to a no longer existing file.

update the build fingerprint

Change-Id: I445c6f86cd358f48aec5ad64a3ef25b31328f743

bieltura committed:
Fix event to enter recovery

This should match the new 4.3 event modified by SONY and already added to msm8960t kernel

Change-Id: If8a20296d04ad5a25800d7e7439ca8afa5baa3eb

brinlyau committed:
huashan: update: audio_policy for cleanup and fixing

Add in back microphone and VOIP calls should work/hangout audio :)
(Thanks @bagyusz).

Change-Id: I8abc8dcca61a07bb2b8c8d67050045618e0a32ba

stellirin committed:
huashan: Allow to use own Adreno blobs

Change-Id: Ie4535a88c022fc65704f76453c12f03ff1feaf64

stellirin committed:
huashan: update sepolicy

Fixes GSM radio and other denials in audit.log

Change-Id: I95b2896cafa6f6b4df1c1f7d5cd5be8adee2587c

stellirin committed:
huashan: remove QCOM_BSP_CAMERA_ABI_HACK

Fuck you, and never come back

Change-Id: I55108aecd43afc5cc2415304f6ddb7c4353d5015

Commit:
Adreno driver cleanup

* Keep only the current driver in qcom-common and move
  the legacy / not latest versions to the platforms needing it.
* Update makefile call / declare supported OpenGL ES version
  appropriately.

Change-Id: I7b70580638083840aaefb2aee2960218525eb3a5

brinlyau committed:
huashan: Remove unnecessary LTE overlay option

already defined in android_vendor_cm

http://review.cyanogenmod.org/#/c/66329/

Change-Id: Ie499da055de61072e587f2b8b20acd8c86914141

brinlyau committed:
huashan: cleanup overlay that is already defined in vendor_cm

Change-Id: I31d4a7c7155fb1c95067086d1d5654ccd547e1eb

brinlyau committed:
huashan: cleanup boardconfig

TARGET_ARCH_VARIANT_CPU is now only used in ARMV6. Let's nuke it

Change-Id: Id22caa74a77a993b8253fdcb23fc4ebbc78d62e4

stellirin committed:
huashan: Remove WiFi config, already in vendor

Change-Id: I58490897152be94ca04092291713f2089684bc5e

stellirin committed:
huashan: Unify with blue-common kernel

One big happy family! <3 *hugs* <3

Change-Id: Ib9a50bc1ba2cde97a633615520ff552edbab83ac

stellirin committed:
Don't DMCA me bro!

Let's remove the useless config files that aren't even used by
Android but accidentally a little bit caused CyanogenMod to make
international headlines because Qualcomm decided it would be a
good idea to allow another company to send out random DMCA take
down requests on their behalf to punish open source projects.

P.S. can I have my repo back now plz.

Change-Id: Iad37025aeea187a2962bd8bfc0588e7ab44bcfde

brinlyau committed:
huashan: we need to sync the right kernel repo

jenkins (and home builders) does need to look for the right kernel repo.
we just forgot to change this when we switched over to the blue-common kernel.

Change-Id: I4819dda1063e840b136d674cf1098a2595ffbd31

stellirin committed:
huashan: audio: Remove paths unknown to CyanogenMod

Change-Id: Ifdab71153090d684ef6e7134875b0f5298e186e5

stellirin committed:
huashan: audio: Rename SGLTE to Voice2

Change-Id: I464677ab3d93df0f55f31e464d063d8f248dbb06

brinlyau committed:
huashan: remove FMradio and cleanup huashan.mk

I'll +1 when it's cleanup up better.

Change-Id: I1205580416c24767c197fa934f9a6b11db24c3bc

mikeNG committed:
Rename lights and camera HAL to use hw_get_module_by_class

Change-Id: I9ac5f74082a3aa7b770362b84ef7fbd0e06edf81

stellirin committed:
huashan: audio: enable fluence audio

* switch to digital microphone
* enable new csd client
* add voice recognition path
* modify "Line", "Speaker DMIC Stereo" and others to
  replace DMIC with ADC

Change-Id: I3fe368b5b4e75001951d2d78714ac2157c35e983

brinlyau committed:
huashan: fixup Dual MIC in video recording recording.

Open source audio with mic recording

Big props to PeterS and Artur for working together with me on this :)
Thanks to Pavol Kucera for the tips

Signed-off-by: Brinly Taylor <uberlaggydarwin@gmail.com>
Change-Id: I523d0c4b714a34191289462c978cfb04a1c2844c

stellirin committed:
Revert "huashan: fixup Dual MIC in video recording recording."

This reverts commit b564574fe91520657ba37e3c738900fcaff60ca3.

Change-Id: I66efff6b23f875f7337461d0e32a25f670589597

Commit:
recovery: remove empty postrecoveryboot script

Change-Id: Ie6993ac800182256dffe0186a8c03dae2a54b904

brinlyau committed:
huashan: fixup buggy mic (add a missing flag)

looks like we missed a flag
When a person switches the call into loudspeaker mode,
suddenly alsa starts using rear mic as "main" mic,
 and it pretty much renders it unusable if you hold the phone the usual way.

this fixes it :) - needs a little more testing

Props to @peters. and @MrSteve555 for informing me about this.

Change-Id: I5ee16c0c31156e3b4d1e65621bb032d35ee1ad6f

Commit:
init: Add setup_fs service for Userdata

* Used for formatting data after unlock,
  as the userdata partition is zeroed out after unlock,
  but a filesystem isn't created.

Change-Id: I17aba9ed9778431e48f9605d10a7fde8be9b7b86

Ultrax5 committed:
huashan: fixup remote submix input device

Change-Id: I07f9c1d3d6a18148377b34a1cb001db075bf5cd5

Ultrax5 committed:
huashan: enable CM performance profiles in device overlay

Change-Id: Ib59c602ed43684f131e3f88bb0464a1938afc192

Ultrax5 committed:
huashan: set readahead during early-init

Change-Id: Ida32fe990bdb5bb83928966fc059ea65e9e14e46

Commit:
huashan: move libqc-opt.so

Change-Id: Ie0a008648008ae946892fa949809bc0dd9357d2c

Commit:
huashan: reorganize cm.deps

Change-Id: Idbd5efef8adf68de2a968885640e69c22bef734a

stellirin committed:
huashan: Use Legacy PAP, our RIL is from 4.3

DCHSPAP 18 -> 30, not 19 -> 30 like 4.4 RIL

Change-Id: I6cd7b18f17791577ae45e4394473b5de4d4b9e1f

Commit:
Inherit from msm8960-common

* msm8960-common then inherits from qcom-common.
* This doesn't change anything currently, just adds one more
  common tree in between - however, later on, we can start moving
  things to this new tree as needed, from both huashan
  and qcom-common

Change-Id: Ib0a5bca4569501e8504ae18b04c98f59130fe4be

Ultrax5 committed:
huashan: fixup the perf profiles

Change-Id: Iffb64cae0edf8d7f448c94265f9d4900c6914951

Commit:
script: fix traditional sort

Change-Id: I23fe22b74f9ac8cd6a09d932f1f4493850c6e7d2

Commit:
script: make osx devs happy

Change-Id: I5fbefbfdc61f575acc2e18d2895498eed918845b

brinlyau committed:
huashan: update build description

Change-Id: I88aad1697ce30f7838381e9f5579cce43c5814af

brinlyau committed:
huashan: cleanup boardconfig.mk

Change-Id: Ieab52472bfdcaef4f5c9cc73c9c19c31f7838321

brinlyau committed:
huashan: cleanup audiopolicy

* match blue-common
* update vendorsetup.sh to build userdebug

Change-Id: I261ebf09f7d11b9dd53739c8a15d4de075f29d0b

brinlyau committed:
huashan: cleanup overlay

match blue-common

Change-Id: I1d42f6cc8722c78d802b3908527a5883864f2a2f

brinlyau committed:
huashan: fixup README pic

Change-Id: I9d83270082d6827ffef5637a94af892e0cac8871

brinlyau committed:
huashan: Enable FM

* Set fm radio to use internal antenna
* add fm flag

Change-Id: I7c9a67371c775a343345b035c196152bd9b2e13d

Mrcl1450 committed:
huashan: Increase volume

Original volume is too low.

Change-Id: I334b4d2e35a606d1fe5440257517560c6afe7d75

cm-gerrit commented:
Hi there;

This is an automated reply, to let you know CyanogenMod doesn't merge pull
requests submitted directly to our github.com repositories.

You can submit your change requests through our public review system,
located at http://review.cyanogenmod.org/

That review system is an installation of Google's Gerrit code review.
If you've never interacted with Gerrit before, we also suggest you take
a look at http://wiki.cyanogenmod.org/wiki/Howto:_Gerrit

Thanks!
